File: customer/Ordered.py

from AbstractEvent import AbstractEvent
import json

class Ordered(AbstractEvent):
    id : int
    item : str
    colorCode : str
    size : str
    quantity : int
    
    def __init__(self):
        super().__init__()
        self.id = None
        self.item = None
        self.colorCode = None
        self.size = None
        self.quantity = None

File: app/src/main/java/shoesdelivery/domain/Ordered.java
package shoesdelivery.domain;

import shoesdelivery.domain.*;
import shoesdelivery.infra.AbstractEvent;
import java.util.*;
import lombok.*;

@Data
@ToString
public class Ordered extends AbstractEvent {

    private Long id;
    private String item;
    private String colorCode;
    private String size;
    private Integer quantity;

    public Ordered(Order aggregate){
        super(aggregate);
    }
    public Ordered(){
        super();
    }
}
